indico/modules/events/surveys/templates/management/survey.html

```diff
@@ -21,7 +21,12 @@
                 {% trans %}Edit settings for this survey.{% endtrans %}
             </div>
             <div class="toolbar">
-                <a href="{{ url_for('.edit_survey', survey) }}" class="i-button icon-edit">
+                <a href="#" class="i-button icon-edit"
+                   data-href="{{ url_for('.edit_survey', survey) }}"
+                   data-title="{%- trans %}Edit survey{% endtrans -%}"
```

Comment:
No need for whitespace trimming. Just do this:
`{% trans %} ... {% endtrans %}`